docs/docs/external-plugins.md

```diff
@@ -241,15 +305,53 @@ bash-3.2$ echo "$?"
 0
 ```
 
+### Tuple Value
+
+`exec`'s value can only be used to specify using a built-in _transport_ provided by Wash. The method tuple must be
```

Comment:
We should mention that `exec`'s tuple-value represents an implementation (currently the only possible implementation is provided by the _transport_ package). That (important) fact is implicit here.

I think the first sentence could be replaced by something like

> `exec`'s value represents an implementation of `exec`. Wash will use this implementation to handle all `exec` calls, so you do _not_ have to implement `exec`'s plugin script invocation for this entry.

> Currently, only implementations provided by the _transport_ package are supported.  The method tuple must be \<rest of docs\>